tidb-binlog/binlogctl/nodes.go

```diff
@@ -63,7 +70,15 @@ func updateNodeState(urls, kind, nodeID, state string) error {
 		if n.NodeID != nodeID {
 			continue
 		}
-		return registry.UpdateNode(context.Background(), node.NodePrefix[kind], n.NodeID, n.Addr, state)
+		switch state {
+		case "pause", "close":
```

Comment:
for "close" you want offline actually? 
how to change the state in etcd if the pump is down?
when will you need to set pausing online closing direcly on etcd?

Reply:
pause and close is used to tell node to exit itself, and then node's status will be paused or offline, the status will changed by node itself. if node is down, this can't work

other state we update the status in etcd directly. In fact online/closing/pausing is useless,  but keep them dose't matter